compose/service.py

```diff
@@ -679,7 +679,8 @@ def labels(self, one_off=False):
 
     def can_be_scaled(self):
         for port in self.options.get('ports', []):
-            if ':' in str(port):
+            external = split_port(port)[1]
+            if external and not (type(external) == tuple and external[1] == None):
```

Comment:
> then checking it for truthiness is unnecessary

You mean `external[1] == None` is not required? I think it is, `external[1]` corresponds to host port, if its specified `can_be_scaled` should fail, else succeed.